Replicator/Puller.cc

```diff
@@ -340,6 +340,15 @@ namespace litecore { namespace repl {
     }
 
 
+    void Puller::_revReRequested(IncomingRev * inc) {
```

Comment:
Parameter should be `Retained<IncomingRev>`, to ensure it isn't freed before the async call occurs.